### PR TITLE
Full width data components

### PR DESCRIPTION
**Pull request by demenech**
**Ticket:** https://github.com/datopian/datahub-next/issues/84

## Changes

- Added `applyFullWidthDirective.tsx` util function
  - This can be applied to any data component to make the `fullWidth` property available for a given component
- Data components that now support the `fullWidth` property
  - Vega (created a new component that wraps the Vega component from `react-vega`)  
  - VegaLite (created a new component that wraps the VegaLite component from `react-vega`)  
  - FrictionlessView
  - Image (new component)
  - Table
- New test repo: https://github.com/datahubio/test-data-repo-2
  - Basically a clone of finance-vix
  - Added examples of all the data components on the README

### CODE REVIEW

**Review by rufuspollock — APPROVED** [review]
Great as usual. Preview is here https://datahub-next-git-feature-fullwidth-components-datopian1.vercel.app/@datahubio/test-data-repo-2
